src/Data Entry/Slider/Basic.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import cn from 'classnames';
+import { Slider } from 'antd';
+
+import './style.less';
+
+
+
+export default (props) => {
+    const {
+        className,
+        ...otherProps
+    } = props;
+
+    const name = cn(className, 'mobile');
+
+    return (
+        <section className="example">
```

Comment:
Please move all story related markup to the stories directory. This component should contain the code for the component and nothing more. 

Only the `<Slider>` should be in this component not section, headings etc. The section, story heading etc should be implemented in stories/LSMobile/Data Entry/Slider/Basic.js which will then use this component.